src/Plugin_Command.php

```diff
@@ -952,7 +955,20 @@ public function is_active( $args, $assoc_args = array() ) {
 	 *     Success: Deleted 1 of 1 plugins.
 	 */
 	public function delete( $args, $assoc_args = array() ) {
+        $all = Utils\get_flag_value( $assoc_args, 'all', false );
+
+        // Check if plugin names of --all is passed.
+        if ( ! ( $args = $this->check_optional_args_and_all( $args, $all, 'delete' ) ) ) {
+            return;
+        }
+
 		$successes = $errors = 0;
+
+        $plugins = $this->fetcher->get_many( $args );
```

Comment:
Lines 967-970 are not needed, this is already covered by the code immediately following.

Just removing these should fix the broken tests.